### ISSUE TITLE
A recommended method for macOS / OSX

### ISSUE THREAD

**Issue by spectrocat** [jaakkopasanen/AutoEq#103]
I have found eqmac2 to be unstable, producing severe distortion after some time of use. I also could not get the Apple AU Lab to work. 

I now use soundflower + Hosting AU on macOS mojave for EQ. Both are free: 

- Soundflower provides the system routing, available by `brew cask install soundflower`

- [Hosting AU](http://ju-x.com/hostingau.html) is a super lightweight DAW that is easy to use with any AU plugin. I use the 10- or 31-band "Apple AUGraphicEQ" that comes with macOS. 

Simply send the system audio to Soundflower:
<img width="201" alt="image" src="https://user-images.githubusercontent.com/58907114/70862541-01643800-1f3e-11ea-8396-30efab452a2d.png">

Then in Hosting AU, make Soundflower the input, stick your EQ as an effect on the Master, and make External Headphones the output: 
<img width="308" alt="image" src="https://user-images.githubusercontent.com/58907114/70862559-34a6c700-1f3e-11ea-863e-2df7bf726bc4.png">

**Comment by jaakkopasanen** [comment]
Hi! Thanks a lot for the suggestions. I'll add these to the OSX section.

**Comment by gustakasn0v** [comment]
Thanks for posting this!

One of the issues I've found when choosing External Headphones in the output, Hosting AU chooses another audio device when the headphones are disconnected. I think this has to do with the audio device disappearing when the headphones are unplugged. Have you found a workaround for this? I tried creating an Aggregate device and a multi-output device in the Audio MIDI setup, but no dice
